src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs

```diff
@@ -1351,7 +1351,11 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Autom
                                     if ($ErrorView -eq 'ConciseView') {
                                         $recommendedAction = $_.ErrorDetails.RecommendedAction
                                         if (-not [String]::IsNullOrWhiteSpace($recommendedAction)) {
-                                            $recommendedAction = $newline + '  Recommendation: ' + $recommendedAction
+                                            $recommendedAction = $newline +
+                                                ${errorColor} +
```

Comment:
Suggestion in line with my other comment is that instead of `${errorColor}` this could be `${recommendationColor}` to allow for differing colours to be used instead.

Reply:
The WG Interactive group has reviewed this.  We agree to implement as this pr describes using ErrorColor.  We would recommend making this customizable in the future based on Recommended Action.

Reply:
Yup, all makes sense to me!